src/dctkit/dec/vector.py

```diff
@@ -22,14 +22,29 @@ def __init__(self, S: spx.SimplicialComplex, is_primal: bool,
         self.coeffs = coeffs
 
 
+class DiscreteTensorField():
+    def __init__(self, S: spx.SimplicialComplex, is_primal: bool,
+                 coeffs: npt.NDArray | Array):
+        self.S = S
+        self.is_primal = is_primal
+        self.coeffs = coeffs
+
+
 class DiscreteVectorFieldD(DiscreteVectorField):
```

Comment:
A "discrete vector field" is a "discrete tensor field" of rank 1. We can make discrete vector field (and its dual variant) a subclass of discrete tensor field.